staff/jclotet/ejercicios/ej2.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+//translate()
+//Write a function translate() that will translate a text into "rövarspråket". 
+//That is, double every consonant and place an occurrence of "o" in between. 
+//For example, translate("this is fun") should return the string "tothohisos isos fofunon".
+
+function translate(word) {
+  var vowel = ['a', 'e', 'i', 'o', 'u', ''];
+  var arrayWords = [''];
+  for (var i=0; i<word.length; i++) {
+    var count = 0;
+    for (var j=0; j<vowel.length; j++) {
+      if (word[i] !== vowel[j]) {
```

Comment:
remember that explained a function called `isVowel`? You can just use this function here